### DISCUSSION TITLE
Barra esperienza

### DISCUSSION THREAD

**Discussion by ziomarlbo** [chromiecraft/chromiecraft#7214] [General]
Buongiorno a tutti, ho ripreso in mano un vecchio pg di liv. 39, ho cercato ovunque per risolvere il problema ma nulla, non riesco a vedere la barra dell'esperienza. Ho altri 2 pg fatti in seguito nelle stesso account e li riesco a vederla, ma su questo nulla, da cosa puo' dipendere? Grazie

**Comment by BeefSteaker** [comment]
Try to enable experience at one of these two
https://www.wowhead.com/wotlk/npc=35365/behsten
https://www.wowhead.com/wotlk/npc=35364/slahtz

**Reply by ziomarlbo** [reply]
Ti ringrazio, ha funzionato perfettamente. Grazie ancora ;)
